framework/source/class/qx/ui/form/AbstractField.js

```diff
@@ -560,28 +560,16 @@ qx.Class.define("qx.ui.form.AbstractField",
       if (this.__oldInputValue && this.__oldInputValue === value) {
         fireEvents = false;
       }
-
-      // check for the filter
-      if (this.getFilter() != null)
+      if (fireEvents)
       {
```

Comment:
@hkollmann could you please explain why the if condition changed from `if(this.getFilter() != null)` to `if(fireEvents)`?